test/unit_ExponentialMap.jl

```diff
@@ -7,7 +7,7 @@ for N in [Float64] # TODO Float32
 
     # sparse matrix
     m = sprandn(n, n, p)
-    m = convert(SparseMatrixCSC{N,Int}, m)
+    m = convert(SparseMatrixCSC{N,Int}, m) # ?
```

Comment:
I do not remember exactly, but there was a problem that the matrix did not have the specific type we needed below.

Reply:
ok. the test work now, i guess it was for some older part..